cicd/gitops/argo-cd-custom-resource-properties.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+[id="argo-cd-custom-resource-properties"]
+= Argo CD Operator
+include::_attributes/common-attributes.adoc[]
+:context: argo-cd-custom-resource-properties
+
+toc::[]
+
+[role="_abstract"]
+The `ArgoCD` resource is a Kubernetes Custom Resource (CRD) that describes the desired state for a given Argo CD cluster and allows for the configuration of the components that make up an Argo CD cluster.
```

Comment:
I noticed the following terms used interchangeably in this PR:

_The `ArgoCD` resource_ (in this file)
_The ArgoCD Custom Resource_ (in `modules/gitops-argo-cd-properties.adoc`)